ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js

```diff
@@ -1442,15 +1577,22 @@
         data.akhMornCount++;
       },
       infoText: function(data) {
-        return 'Akh Morn #' + data.akhMornCount;
+        return {
+          en: 'Akh Morn #' + data.akhMornCount,
+          cn: '死亡轮回 #' + data.akhMornCount,
+        };
       },
       tts: function(data) {
-        return 'akh morn ' + data.akhMornCount;
+        return {
```

Comment:
Can you remove this `tts` block entirely? This is old, and is trying to fix the fact that `#` wasn't getting called out correctly.  I've fixed it so that tts automatically just drops those, so this should call out just fine without the `tts` block.

Reply:
removed